package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "setup-dbs": "psql -f ./db/setup.sql",
     "seed": "node ./db/seeds/run-seed.js",
-    "test": "jest",
+    "test": "npm run setup-dbs && npm run seed && jest",
```

Comment:
I dont understand why this script has been updated were you having problems with jest handling calling the seed function. As it stands your script will drop both the test and dev database and reseed the dev database each time you your tests, I dont know if this is behaviour that you want